src/main/java/com/esri/samples/localserver/local_server_dynamic_workspace_shp/README.md

```diff
@@ -0,0 +1,42 @@
+<h1>Local Server Dynamic Workspace Shapefile</h1>
+
+<p>Demonstrates how to dynamically add a local shapefile to a map using the Local Server. </p>
+
+<p><b>Note:</b> Local Server is not supported on MacOS</p>
+
+<img src="LocalServerDynamicWorkspaceShapefile.png"/>
+
+<h2>How to use the sample</h2>
+
+<p>A Local Server and Local Map Service will automatically be started and once running a Feature Layer will be created and added to the map. </p>
+
+<h2>How it works</h2>
+
+<p>To create a <code>ShapefileWorkspace</code> and add it to a <code>LocalMapService</code>:</p>
+
+<ol>
+<li>Create and run a local server.
+<ul><li><code>LocalServer.INSTANCE</code> creates a local server</li></ul>
+<li><code>Server.startAsync()</code> starts the server asynchronously</li></ul>
+<li>Create a <code>LocalMapService</code> instance using an empty .MPK file (the sample uses one that is created for you). Don't start it yet.</li>
+<li>Create the <code>ShapefileWorkspace</code> and <code>TableSublayerSource</code> instances.</li>
+<li>Add the <code>ShapefileWorkspace</code> the list of dynamic workspaces of the <code>LocalMapService</code>.</li>
```

Comment:
<li>Add <code>ShapefileWorkspace</code> to  <code>LocalMapService.setDynamicWorkspaces()</code>.</li>